src/components/Tooltip/Tooltip.tsx

```diff
@@ -191,6 +195,7 @@ const Tooltip = ({
   positionElementClassName,
   tooltipClassName,
   zIndex,
+  delay = 500,
```

Comment:
I wonder if 500 is a little too long? Playing around with the number it seemed like 350 was a good amount of time to me.

Reply:
Changed default to 350ms